source/javascripts/components/SplashModal/SplashModal.scss

```diff
@@ -0,0 +1,13 @@
+.ModalHeader {
+	height: 200px;
```

Comment:
Do it with `$grid * 50` instead. You will need to include _constants.scss for this.

Reply:
i'm not a big fan of using magic/global variables in CSS to tie together unrelated things especially with element sizing. the variable name is really non-descriptive

Reply:
The grid is what all our elements need to be the multiple of, regarding width & height. There is probably a Bitkit global for this, it would be better to use thet, I wasn't aware it's accessible here.